sample/bucket.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Bucket
```

Comment:
should be mentioned somewhere that a `Bucket` and `BucketAccess` are typically created by COSI and not by the user or admin (with brownfield exception noted).

Reply:
Added a comment block above each of the manifests in the sample folder